Adding one line comment to procedure template (#224)
redhat-documentation/modular-docs: modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

= Doing procedure A
////
Start the title of a procedure module with a gerund, such as Creating, Installing, or Deploying.

> Be sure to include a line break between the title and the module introduction.

Reviewer: "Include" can come across as "add", which isn't needed in this case.
How about "maintain" or something similar? For example:
```suggestion
Maintain the line break between the title and module introduction.
```

Author: Thanks Rolfe, I used the same line that is already in the other templates for consistency. So  I guess it would be best to leave the line as it is or update the same line in the other templates too.
